go/adbc/driver/databricks/command.go

```diff
@@ -0,0 +1,384 @@
+	executor, err := ce.Start(ctx, cmd.conn.client.Config.ClusterID, compute.LanguageSql)
+	if err != nil {
+		return nil, adbc.Error{
+			Code: adbc.StatusUnknown,
+			Msg:  fmt.Sprintf("[Databricks] failed to start command execution: %s", err),
+		}
+	}
+
+	res, err := executor.Execute(ctx, cmd.req.Command)
+	if err != nil {
+		return nil, adbc.Error{
+			Code: adbc.StatusUnknown,
+			Msg:  fmt.Sprintf("[Databricks] failed to execute command: %s", err),
+		}
+	}
+
+	for {
+		switch res.ResultType {
+		case compute.ResultTypeTable:
+			// CommandExecution API doesn't have a ChunkIndex like StatementExecution
+			return NewCommandRecordReader(ce, cmd.commandId, res)
+		case compute.ResultTypeText:
+			return NewCommandRecordReader(ce, cmd.commandId, res)
+		case compute.ResultTypeError:
+			return nil, adbc.Error{
+				Code: adbc.StatusUnknown,
+				Msg:  fmt.Sprintf("[Databricks] command execution failed: %s", res.Error),
+			}
+		default:
+			return nil, adbc.Error{
+				Code: adbc.StatusInternal,
+				Msg:  fmt.Sprintf("[Databricks] Unexpected command result type: %s", res.ResultType),
+			}
+		}
+	}
+}
+
+func unexpectedCommandState(state compute.CommandStatus) error {
+	return adbc.Error{
+		Code: adbc.StatusInternal,
+		Msg:  fmt.Sprintf("[Databricks] Unexpected command state: %s", state),
+	}
+}
+
+// ExecuteUpdate executes a statement that does not generate a result
+// set. It returns the number of rows affected if known, otherwise -1.
+func (cmd *command) ExecuteUpdate(ctx context.Context) (int64, error) {
+	if cmd.req.Command == "" {
+		return -1, adbc.Error{
+			Code: adbc.StatusInvalidState,
+			Msg:  "ExecuteUpdate called before SetSqlQuery",
+		}
+	}
+
+	// For CommandExecution API, we can use ExecuteCommand and check if it returns a result
+	ce := cmd.conn.CommandExecution()
+	executor, err := ce.Start(ctx, cmd.conn.client.Config.ClusterID, compute.LanguageSql)
+	if err != nil {
+		return -1, adbc.Error{
+			Code: adbc.StatusUnknown,
+			Msg:  fmt.Sprintf("[Databricks] failed to start command execution: %s", err),
+		}
+	}
+	res, err := executor.Execute(ctx, cmd.req.Command)
+	if err != nil {
+		return -1, adbc.Error{
+			Code: adbc.StatusUnknown,
+			Msg:  fmt.Sprintf("[Databricks] failed to execute command: %s", err),
+		}
+	}
+
+	// If the command has a result, it's a query, not an update
+	if res.Data != nil {
+		return -1, adbc.Error{
+			Code: adbc.StatusInvalidState,
+			Msg:  "ExecuteUpdate called with a query that returns results",
+		}
+	}
+
+	// For updates, we don't have a row count in the CommandExecution API
+	return -1, nil
+}
+
+// Prepare turns this statement into a prepared statement to be executed
+// multiple times. This invalidates any prior result sets.
+func (cmd *command) Prepare(ctx context.Context) error {
+	// CommandExecution API doesn't support prepared statements
+	return adbc.Error{
+		Code: adbc.StatusNotImplemented,
+		Msg:  "Prepare not yet implemented for Databricks CommandExecution API",
+	}
```

Comment:
Make it a no-op instead so we can call `prepare()` from the Rust side generically.

```suggestion
	return nil
```